addons/project/views/project_views.xml

```diff
@@ -184,10 +184,10 @@
                             </group>
                         </page>
                     </notebook>
+                    <div class="oe_chatter" groups="base.group_user">
```

Comment:
I don't get why you moved `groups` from `<field>` to `<div>`.
In fact, it looks weird to hide this for unauthenticated users, since this view looks visible only by authenticated users in the first place... Or maybe I miss something?

If there's no motivation to move it, I would play safe and keep it on `<field>`